File: VB/Converter.vb
Imports System
Imports System.Windows.Data
Imports DevExpress.Xpf.Docking

Namespace Example2
    Public Class Converter
        Implements IValueConverter

        Public Function Convert(ByVal value As Object, ByVal targetType As Type, ByVal parameter As Object, ByVal culture As System.Globalization.CultureInfo) As Object Implements IValueConverter.Convert
            Dim item As BaseLayoutItem = TryCast(value, BaseLayoutItem)
            Dim model As MainViewModel = TryCast(parameter, MainViewModel)
            Return value.ToString()

        End Function
        Public Function ConvertBack(ByVal value As Object, ByVal targetType As Type, ByVal parameter As Object, ByVal culture As System.Globalization.CultureInfo) As Object Implements IValueConverter.ConvertBack
            Dim name As String = value.ToString()
            Dim model As MainViewModel = TryCast(parameter, MainViewModel)
            Return name
        End Function
    End Class
End Namespace

File: CS/Converter.cs
using System;
using System.Windows.Data;
using DevExpress.Xpf.Docking;

namespace Example2 {
    public class Converter : IValueConverter {
        public object Convert(object value, Type targetType, object parameter, System.Globalization.CultureInfo culture) {
            BaseLayoutItem item = value as BaseLayoutItem;
            MainViewModel model = parameter as MainViewModel;
            return value.ToString(); ;
        }
        public object ConvertBack(object value, Type targetType, object parameter, System.Globalization.CultureInfo culture) {
            string name = value.ToString();
            MainViewModel model = parameter as MainViewModel;
            return name;
        }
    }
}